MaxText/pyconfig.py

```diff
@@ -17,6 +17,7 @@
 # pylint: disable=missing-module-docstring
 from collections import OrderedDict
 
+import hardware_map
```

Comment:
I mean reading this it is obviously the wrong name for the file. hardware_map could be anything! How about ```accelerator_to_spec_map```?

Reply:
Done.